com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -517,13 +517,67 @@ private void Update()
                 {
                     ;
                 }
+                
+#if MULTIPLAYER_TOOLS
+                if (NetworkManager.Singleton.IsServer)
+                {
+                    var ngoConnectionIds = NetworkManager.Singleton.ConnectedClients.Keys;
+                    foreach (var ngoConnectionId in ngoConnectionIds)
+                    {
+                        ExtractNetworkMetrics(NetworkManager.Singleton.ClientIdToTransportId(ngoConnectionId));
+                    }
+                }
+                else
+                {
+                    ExtractNetworkMetrics(NetworkManager.Singleton.ClientIdToTransportId(NetworkManager.Singleton.ServerClientId));
+                }
+#endif
             }
         }
 
         private void OnDestroy()
         {
             DisposeDriver();
         }
+        
+#if MULTIPLAYER_TOOLS
+        private void ExtractNetworkMetrics(ulong transportClientId)
+        {
+            var networkConnection =  ParseClientId(transportClientId);
+            
+            //Two pipelines, unreliable sequence and reliable fragmented
+            for (byte i = 0; i < 2; ++i)
```

Comment:
Should `2` be a const instead of leaving the comment?

If there's only 2 pipelines, should this just be two explicit function calls and pass in the pipeline as a parameter? The for loop seems kind of unnecessarily complex control flow since we have a very small and const-size set of pipelines that aren't actually in a collection that we can iterate over.